src/wrapped_radix_engine_toolkit/low_level/signed_intent.rs

```diff
@@ -1,25 +1,72 @@
 use crate::prelude::*;
 
+pub use radix_engine_toolkit::functions::signed_intent::compile as RET_signed_intent_compile;
+
 #[derive(Clone, Debug, PartialEq, Eq, uniffi::Record)]
 pub struct SignedIntent {
-    pub intent: TransactionIntent,
+    intent: TransactionIntent,
```

Comment:
Hide constructor so that we ensure we always go via `new` which now is a failable constructor, performing `compile_signed_intent_with ` - checking SBOR depth.